lightkurve/correctors/pldcorrector.py

```diff
@@ -179,12 +179,28 @@ def create_design_matrix(self, pld_order=3, pca_components=16, pld_aperture_mask
 
         # We set the width of all coefficient priors to median flux divided
         # by 3 to prevent the fit from going crazy.
-        prior_sigma = np.nanmedian(self.lc.flux.value)
+        prior_sigma = np.nanstd(self.lc.flux.value) * 10
```

Comment:
Can you expand on why you think `nanstd` is better than `nanmedian`?  My thinking was that the components of the designmatrix all tend to be normalized (because `pca()` normalizes them), so my reasoning was that we want to allow coefficients to range "roughly between zero and the median flux".

Do we think it is more principled to use `nanstd`?  Would this not  prevent a fit if the median flux is very large compared to the scatter?  Perhaps we could allow for that by setting `prior_mu` of the `append_constant` column equal to the median flux?